src/serial.rs

```diff
@@ -141,24 +186,31 @@ macro_rules! hal {
                     Serial { usart, pins }
                 }
 
+                /// Starts listening by enabling _RX buffer not empty_ interrupt and _TX register
+                /// empty_ interrupt
```

Comment:
"RX buffer not empty" does not sound right. The interrupts are just "tx and rx interrupts"

Reply:
I now use the names from the datasheet:
- Transmit data register empty
- Received data ready to be read